processor/src/stack/mod.rs

```diff
@@ -66,6 +66,32 @@ impl Stack {
         Ok(self.trace[0][self.step])
     }
 
+    /// Get states from the stack, which includes both the
+    /// trace state and overflow table.
+    /// If n is not passed in, this returns all states.
+    pub fn get_states(&self, n: Option<usize>) -> Vec<Felt> {
```

Comment:
Should we call this `get_values()` to keep it consistent with methods in `Memory` struct?